### DISCUSSION TITLE
Deployment stuck in Queued

### DISCUSSION THREAD

**Discussion by AnmolGarg14** [vercel/community#3930] [Help] [answered]
### Summary

So, I was trying to deploy my project into Vercel with github and its stuck now in that Queue thing I dont know what to do, like do I just wait till it works?

URL: https://admindashboard-btwsdpg5d-anmolgarg14.vercel.app

### Steps to Reproduce

Deployment in queued

**Comment by swarnava** [comment, marked as answer]
Hi,

I can see that the deployment is successfully built now.
